config/detekt/detekt.yml

```diff
@@ -77,7 +77,7 @@ comments:
     searchProtectedProperty: false
   OutdatedDocumentation:
     active: true
-    excludes: [ '**/test/**', '**/androidTest/**', '**/demoapp/**', '**/api/models/**', '**/restapi/models/**' ]
+    excludes: [ '**/test/**', '**/androidTest/**', '**/demoapp/**', '**/restapi/models/**', '**/restapi/apis/**' ]
```

Comment:
Detekt was complaining about `OutdatedDocumentation` but we have no control over those files, so added `apis` as well here.

Reply:
🤔  I wonder why it hasn't complained before.

In fact, it's just:

```
gravatar/src/main/java/com/gravatar/restapi/apis/AvatarsApi.kt:73:17: Documentation of uploadAvatar is outdated [OutdatedDocumentation]
```

Not sure why it's only that method.

Reply:
yeah, but still we don't control that file, so I think it's valid to exclude it.

Reply:
Yeah, just trying to understand the reason behind the error.